beacon.el

```diff
@@ -246,6 +246,10 @@ COLORS applied to each one."
   ;; The after-string must not be longer than the remaining columns
   ;; from point to right window-end else it will be wrapped around.
   (let ((colors (seq-take colors (- (window-width) (beacon--visual-current-column) 1))))
+    (if (and (derived-mode-p 'org-mode)
+             (org-invisible-p))
```

Comment:
Please extract this to a function `beacon--inside-org-folded-p`, and wrap it in `ignore-errors` just so we don't break if org ever changes or removes that function.